src/app/database/__init__.py

```diff
@@ -157,23 +162,32 @@ def populate_db():
     gender_description = {"male": "Masculino", "female" :"Feminino"}
 
     cities_db_data = City.query.order_by(City.name.asc()).all()
-    cities_db_dict = cities_share_schema.dump(cities_db_data)
+    genders_db_data = Gender.query.all()
 
     for user in users_data_request.json()['results']:
         if user['gender'] in gender_description:
             gender = gender_description[user['gender']]
-        for city in cities_db_dict:    
-            if city['name'] == user['location']['city']:
-                User.seed(
-                gender_id = gender,
-                city_id= city['id'],
-                name = user['name']['first'] + ' ' + user['name']['last'],
-                age = user['dob']['date'],
-                email = user['email'],
-                phone = user['cell'],
-                password = gera_password(),
-                street = user['location']['street']['name'],
-                number_street = user['location']['street']['number']
-                )
+        for gen in genders_db_data: 
+            if gen.description == gender:
+                gender_id = gen.id
```

Comment:
O loop for para city_id não precisa estar dentro do loop for para gender_id, correto?

Reply:
Se for acrescentado o "else" lá em cima, precisa inserir o gênero "outros" no banco de dados, fazendo `genders_descriptions = ["Masculino", "Feminino", "Outros"]`